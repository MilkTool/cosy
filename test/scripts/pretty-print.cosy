var b=3{
var a=b+4
fn g(){return fn(x){return(a+x)*2}}print g()(2)}
// for(var i=0;i<10;i=i+1){fn g(){return fn(x){return(a+x)*2;};}print g()(2);}
// var a=3;{var a=4;print fn(){return fn(x){return(a+x)*2;};}()(2);}
