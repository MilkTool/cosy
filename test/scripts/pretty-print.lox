var a=3;{var a=4;fun g(){return fun(x){return(a+x)*2;};}print g()(2);}
// for(var i=0;i<10;i=i+1){fun g(){return fun(x){return(a+x)*2;};}print g()(2);}
// var a=3;{var a=4;print fun(){return fun(x){return(a+x)*2;};}()(2);}
