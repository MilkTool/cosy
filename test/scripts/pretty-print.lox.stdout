var a = 3;
{
  var a = 4;
  fun g() {
    return fun (x) {
      return (a + x) * 2;
    };
  }
  print g()(2);
}
